### DISCUSSION TITLE
[Request] Config options to disable explosive tank sound and flame fx

### DISCUSSION THREAD

**Discussion by MathiusXII** [EtherealEmil/UsualScrap#14] [Ideas]

**Comment by EtherealEmil** [comment]
After I figure out how to even do configs I will definitely add an option to disable sounds that loop. I could do the flame effect but I'd like to know why you'd want it off, does it cause performance issues? I was  that I was also planning to make the sound and effect stop after being brought to the ship but just haven't gotten to it yet.

**Comment by MathiusXII** [comment]
I just personally thought that if the tank was leaking and on fire that it would've run out of gas or exploded long before I arrived to smack it with a shovel.

Just a personal nitpick.

**Comment by MathiusXII** [comment]
Btw do you think you could possibly add functionality to other mod's scraps? I really like the looks of VanillaContentExpansion's scraps, but the author seems to have gone mia and hasn't added functionality to them.

**Comment by EtherealEmil** [comment]
I definitely could but giving functionality to scrap isn't the simplest thing to do in general and making it work while patching other mods would probably be a nightmare so I probably wouldn't bother, it's much easier to just create scrap items entirely in one mod as I have easy access to everything about them without patching.

**Comment by MathiusXII** [comment]
Understandable. I would try myself but I couldn't find a link to his source code and I'm a noob at modding

**Comment by EtherealEmil** [comment]
If you look at the mod on the thunderstore website you can go to the source tab and see some of the code, for large mods you have to manually download them and maybe use a program like asset ripper or dnspy to view the code, just make sure it's license allows whatever you may do. Modding takes some time but isn't too bad, I knew absolutely nothing about coding or modding 2 months ago.
